docs/README.md

```diff
@@ -1,3 +1,3 @@
-# About Thunderbird for Android
+# Thunderbird for Android Documentation
 
-This document provides an overview of the Thunderbird for Android project.
+The latest available documentation is rendered at: [https://thunderbird.github.io/thunderbird-android/docs/latest/](https://thunderbird.github.io/thunderbird-android/docs/latest/)
```

Comment:
This should be part of the main readme, as this file is the landing page for the docs and would render a link to itself.

Reply:
I was wondering about this. I'm trying to avoid the situation where a user stumbles into the docs folder on github, and simply sees the empty README. I know the rendered version would link to itself, but I think this is a tradeoff that is ok to make. What do you think?

Reply:
The current README needs more content, at the moment it's just a placeholder and should transform into a real landing page, guiding where to look for which aspect of the documentation.

But we could remove the README from the SUMMARY and add the landing page as separate document.

Reply:
Let's take care of the landing page a bit later, I don't really have thoughts on content for it yet that wouldn't duplicate the main README. At this time we can also consider if there are aspects we should integrate into DTN.

For now I removed the main README, making the contributing page the landing page, which isn't too bad.